cmd/kube-apiserver/app/options/validation.go

```diff
@@ -45,6 +45,20 @@ func validateServiceNodePort(options *ServerRunOptions) []error {
 	return errors
 }
 
+func validateAuditLogFlags(options *ServerRunOptions) []error {
```

Comment:
The tests look good 👍 

Can you move them into `AuditLogOptions.Validate` in `vendor/k8s.io/apiserver/pkg/server/options/audit.go`? Here are only the kubernetes specific validations. Audit log is in k8s.io/apiserver.